packages/app-project/stores/Store.js

```diff
@@ -1,4 +1,5 @@
-import { getEnv, types } from 'mobx-state-tree'
+import { addMiddleware, getEnv, types } from 'mobx-state-tree'
+import { logNodeError } from '../src/helpers/logger'
```

Comment:
fwiw I think I named this function poorly

Reply:
I’ve renamed it `logToSentry` in #2044.